app/additem.tsx

```diff
@@ -0,0 +1,199 @@
+import { useGlobalSearchParams, useRouter } from "expo-router";
+import React, { useState } from "react";
+import {
+    StyleSheet,
+    Text,
+    TextInput,
+    TouchableWithoutFeedback,
+    Keyboard,
+    KeyboardAvoidingView,
+    Platform,
+    ScrollView,
+    Alert,
+    View,
+    TouchableOpacity,
+} from "react-native";
+import RNDateTimePicker from "@react-native-community/datetimepicker";
+
+const AddItem = () => {
+    const router = useRouter();
+    const { day, date } = useGlobalSearchParams();
+    const [label, setLabel] = useState("");
+    const [description, setDescription] = useState("");
+    const [startTime, setStartTime] = useState(new Date(0, 0, 0, 0, 0)); // Default to 00:00
+    const [endTime, setEndTime] = useState(new Date(0, 0, 0, 23, 59)); // Default to 23:59
+    const [showStartPicker, setShowStartPicker] = useState(false);
+    const [showEndPicker, setShowEndPicker] = useState(false);
+
+    const handleStartTimeChange = (event: any, selectedDate: Date | undefined) => {
+        setShowStartPicker(false);
+        if (selectedDate) {
+            if (selectedDate > endTime) {
+                Alert.alert("Ugyldig tid", "Starttiden kan ikke være senere end sluttiden.");
+            } else {
+                setStartTime(selectedDate);
+            }
+        }
+    };
+
+    const handleEndTimeChange = (event: any, selectedDate: Date | undefined) => {
+        setShowEndPicker(false);
+        if (selectedDate) {
+            if (selectedDate < startTime) {
+                Alert.alert("Ugyldig tid", "Sluttiden kan ikke være før starttiden.");
+            } else {
+                setEndTime(selectedDate);
+            }
+        }
+    };
```

Comment:
Both of these functions first if statement can be inverted to remove mental load.

Reply:
I changed, approved?